src/NexusMods.App.UI/Pages/Downloads/InProgressViewModel.cs

```diff
@@ -471,6 +476,16 @@ private void UpdateWindowInfoInternal()
             _customSeparators.Add(FirstMultiple(throughput));
         }
 
+        const int delay = 5;
+        if (_throughputValues.Count > delay)
+        {
+            _lineSeries.IsVisible = _throughputValues.TakeLast(delay).Sum(x => x.Value!.Value) > 0.0;
+        }
+        else
+        {
+            _lineSeries.IsVisible = false;
```

Comment:
```suggestion
            _lineSeries.IsVisible = _throughputValues.TakeLast(_throughputValues.Count).Sum(x => x.Value!.Value) > 0.0;
```

Otherwise if you start a DL or a dl is ongoing it will not be shown for 5 seconds

Reply:
Or just change the thing to pass in 
`_throughputValues.TakeLast(_throughputValues.Count < 5 ? _throughputValues.Count : 5 )`